sqlglot/optimizer/qualify_tables.py

```diff
@@ -15,26 +15,18 @@ def qualify_tables(
     schema: t.Optional[Schema] = None,
 ) -> E:
     """
-    Rewrite sqlglot AST to have fully qualified, unnested tables.
+    Rewrite sqlglot AST to have fully qualified tables. Join constructs such as
+    (t1 JOIN t2) AS t will be expanded into (SELECT * FROM t1 AS t1, t2 AS t2) AS t.
 
     Examples:
         >>> import sqlglot
         >>> expression = sqlglot.parse_one("SELECT 1 FROM tbl")
         >>> qualify_tables(expression, db="db").sql()
         'SELECT 1 FROM db.tbl AS tbl'
         >>>
-        >>> expression = sqlglot.parse_one("SELECT * FROM (tbl)")
+        >>> expression = sqlglot.parse_one("SELECT 1 FROM (t1 JOIN t2) AS t")
```

Comment:
why did you change this to 1 from *?

Reply:
no important reason, just to be consistent with the previous example in the docstring